README.md

```diff
@@ -145,6 +145,8 @@ RUST_LOG="debug,forest_libp2p::service=info" forest
 
 Will show all debug logs by default, but the `forest_libp2p::service` logs will be limited to `info`
 
+Forest can also expose a [Loki](https://github.com/grafana/loki) endpoint to facilitate logs monitoring. Use `--loki` to enable it and `--loki-endpoint` to specify the interface and the port.
```

Comment:
Maybe more precisely:
`Forest can also send telemetry to the endpoint of a Loki instance or a Loki agent (See [Grafana Cloud](https://grafana.com/oss/loki/))`